src/pages/search/map.tsx

```diff
@@ -44,14 +43,26 @@ export const MAP_SEARCH_QUERY = gql`
       edges {
         node {
           venue {
-            ...unifiedSearchVenueFragment
+            meta {
+              id
+            }
+            name {
+              fi
+              sv
+              en
+            }
+            location {
+              geoLocation {
+                geometry {
+                  coordinates
+                }
+              }
+            }
```

Comment:
I understand the purpose here and I think it's worth it because we are asking for so much data in one go. This does mean that we can't reuse the fetched data in other views, and that other views will have to request the more complete set of data in a new request. It's not worth it to preload additional fields from thousands of results in order to avoid making the additional query though.